homeassistant/components/litejet/light.py

```diff
@@ -91,16 +93,37 @@ def extra_state_attributes(self):
 
     def turn_on(self, **kwargs):
         """Turn on the light."""
+        is_complex = False
+        brightness = 99
+        transition = self._config_entry.options.get(CONF_DEFAULT_TRANSITION, 0)
+
         if ATTR_BRIGHTNESS in kwargs:
+            is_complex = True
             brightness = int(kwargs[ATTR_BRIGHTNESS] / 255 * 99)
-            self._lj.activate_load_at(self._index, brightness, 0)
+
+        if ATTR_TRANSITION in kwargs:
+            is_complex = True
+            transition = kwargs[ATTR_TRANSITION]
+
+        if is_complex:
+            self._lj.activate_load_at(self._index, brightness, int(transition))
         else:
             self._lj.activate_load(self._index)
 
     def turn_off(self, **kwargs):
         """Turn off the light."""
-        self._lj.deactivate_load(self._index)
+        is_complex = False
+        transition = self._config_entry.options.get(CONF_DEFAULT_TRANSITION, 0)
+
+        if ATTR_TRANSITION in kwargs:
+            is_complex = True
+            transition = kwargs[ATTR_TRANSITION]
+
+        if is_complex:
```

Comment:
This logic is weird, so if turn_off is called, without a transition, we are not using the default transition?

Reply:
But in that case, this code makes no sense, as well... the default transition is never used, thus this logic isn't needed at all.

Reply:
Ah ha. You are very right. It kept too much complexity when I copied it from the turn_on implementation. I never noticed this in the year I've been carting this change around in my private branch. I'll simplify.